notebooks/curation_api/python/src/utils/config.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+
+
+def set_api_urls(env: str = "prod"):
+    """
+    This function sets url environment variables that other Curator API notebook modules use when calling
+    Curator API endpoints.
+    :param env: the deployment environment
+    :return: None
+    """
+    if env == "prod":  # For all official use of the cellxgene product
+        domain_name = "cellxgene.cziscience.com"
+    elif env == "dev" or env == "staging":  # For testing purposes only
+        domain_name = f"cellxgene.{env}.single-cell.czi.technology"
+    else:
+        raise Exception("Must provide env arg: 'dev', 'staging', or 'prod'.")
+
+    os.environ["site_url"] = f"https://{domain_name}"
+    print(f"Set 'site_url' env var to {os.getenv('site_url')}")
+    os.environ["api_url_base"] = f"https://api.{domain_name}"
+    print(f"Set 'api_url_base' env var to {os.getenv('api_url_base')}")
+
+
+def set_access_token(api_key_file_path: str):
+    """
+    This function uses the API key to retrieve a temporary access token from the Curator API. It then sets
+    the 'access_token' environment variable, which other Curator API notebook modules use when calling
+    Curator API endpoints. The usage of this function relies on a *prior* set_api_urls() call (from api_urls.py).
```

Comment:
I think if this function relies on a prior set_api_urls() call to work, we may as well just make the call in this function if api_url_base is not set (and have an optional 'env' parameter, defaulting to prod). Then we can keep the set-up to one call in the notebooks as well. 

I like keeping set_api_urls logic separated into a helper function, though.

Reply:
This statement can be removed now.